src/vs/workbench/contrib/notebook/common/model/notebookCellTextModel.ts

```diff
@@ -291,7 +291,7 @@ export class NotebookCellTextModel extends Disposable implements ICell {
 			}
 
 			this.outputs.splice(splice.start + commonLen, splice.deleteCount - commonLen, ...splice.newOutputs.slice(commonLen));
-			this._onDidChangeOutputs.fire({ start: splice.start + commonLen, deleteCount: splice.deleteCount - commonLen, newOutputs: splice.newOutputs.slice(commonLen) });
```

Comment:
The start values were incorrect as a result the previous output items were not disposed.

Reply:
Note: The original change was introduced to fix  https://github.com/microsoft/vscode/issues/132143
With this change I can see some flickering.
I.e. there is some regression with this PR.